sktime/clustering/__init__.py

```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+__all__ = ["TimeSeriesKMeans", "TimeSeriesKMedoids", "TimeSeriesKShapes"]
+
+from sktime.clustering._time_series_k_means import TimeSeriesKMeans
```

Comment:
I'd prefer shorter files names
```suggestion
from sktime.clustering._k_means import TimeSeriesKMeans
```